backends/cuda-ref/ceed-cuda-ref-vector.c

```diff
@@ -509,7 +509,7 @@ static int CeedVectorNorm_Cuda(CeedVector vec, CeedNormType type, CeedScalar *no
           CeedCallCublas(ceed, cublasSnrm2(handle, (CeedInt)sub_length, (float *)d_array_start, 1, &sub_norm));
           norm_sum += sub_norm * sub_norm;
         }
-        *norm            = sqrt(norm_sum);
```

Comment:
looks like the formatter really wants this here

Reply:
my (`clang-format-17`) formatter strongly disagrees

Reply:
I personally disagree, but CI uses 16

Reply:
I'll force it to cooperate

Reply:
that's so weird, there's similar code nearby that formats correctly...

Reply:
Yeah, every so often clang-format hits an edge case.